gaurav/src/components/Log.tsx

```diff
@@ -0,0 +1,160 @@
+import Button from "./ui/personal/Button";
+import Card from "./ui/Card";
+import { useState, useEffect } from "react";
+import Errors from "./ui/Errors";
+import { useLocation } from "react-router-dom";
+// import { useLocation } from "react-router-dom";
```

Comment:
clean up this file and organize code for readability

Reply:
Done